### PR TITLE
Accept database/table in raw destination type

### PR DESCRIPTION
**Pull request by emelvarol-cognite**
UI accidentally had it this way and I thought it makes sense to accept these field names also.

### CODE REVIEW

File: tests/test_config.py [hunk 1]
```diff
@@ -157,6 +150,75 @@ def test_load_config_file_oidc() -> None:
...
+    assert conf.action == ActionType.upsert
+    from cognite.client.data_classes import RawTable
+
+    dest = to_destination(conf.destination)
+    assert isinstance(dest, RawTable)
+    assert dest.database == "testDb"
+    assert dest.table == "testTable"
+    rmdir(Path(test_name))
+
+    # also test rawDatabase and rawTable fields
```
Comment (emelvarol-cognite) [inline tests/test_config.py:202]: @wjoel here 😉

Comment (wjoel) [reply tests/test_config.py:202]: :raised_eyebrow: just add a new file instead.

Comment (emelvarol-cognite) [reply tests/test_config.py:202]: Sure

**Comment by emelvarol-cognite** [comment]
@wjoel can you take a look again?

**Review by wjoel — APPROVED** [review]
Not sure I understand the logic behind putting those examples into strings instead of files as we did previously, maybe makes sense for the "rawDatabase" case as we don't want it to be used as an example, but lgtm.

**Pull request merged**